docs/kms/README.md

```diff
@@ -140,6 +142,21 @@ Please use your own master key. A random master key can be generated using e.g.
 head -c 32 /dev/urandom | xxd -c 32 -ps
 ```
 
+**2.2.2 KMS master key from docker secret**
+
+Alternatively, you may pass a master key as a [Docker secret](https://docs.docker.com/engine/swarm/secrets/).
+
+```bash
+echo "my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574" | docker secret create sse_master_key
+```
+
```

Comment:
It seems obvious but the shown demo key (`6368616e676520746869732070617373776f726420746f206120736563726574`) must not be used for anything real. We should make this explicit.

Maybe `Obviously, do not use this demo key for anything real!`